src/Features/JsonPatch/src/Converters/JsonPatchDocumentConverter.cs

```diff
@@ -13,6 +12,8 @@ namespace Microsoft.AspNetCore.JsonPatch.Converters
 {
     public class JsonPatchDocumentConverter : JsonConverter
     {
+        internal static IContractResolver DefaultContractResolver { get; } = new DefaultContractResolver();
```

Comment:
Any reason not to do this:

```suggestion
        internal static DefaultContractResolver DefaultContractResolver { get; } = new();
```